client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	LinkQuit,
+	makeTour,
+	Quit,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	isEnabled,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const BoostedPostTour = makeTour(
+	<Tour
+		name="boostedPostSurvey"
+		path="/posts/"
+		version="20170127"
+		when={ and(
+			isEnabled( 'boosted-post-survey' ),
+			isDesktop,
+			isAbTestInVariant( 'boostedPostSurvey', 'enabled' )
+		) }
+	>
+		<Step name="init" placement="right">
+			<p>{ translate( 'Interested in increasing your readers?' ) }</p>
+			<ButtonRow>
+				<LinkQuit
+					primary
+					target="_blank"
+					href="http://jonburke.polldaddy.com/s/boosted-post">
```

Comment:
Ah, thanks. Updated to HTTPS. 

We're trying to utilize as many of our pre-existing features as possible for this test, are you aware of another option that feels less artisanal?